platform/podjs/README.md

```diff
@@ -20,21 +20,130 @@ feature. While we could support the creation of a container and a
 mechanism for loading features in `pod.js` without too much trouble,
 it is currently not implemented.
 
-The `polyOut` and `polyNav` interfaces are not implemented at the
-moment. The former because we don't use it yet, the latter because it
-is currently not part of the `Pod` interface, which we aim to rectify
-soon.
+## Prerequisites
+
+Make sure you have already installed the following:
+
+-   `node` `v16` or higher
+-   `npm` `v7` or higher
+-   `rollup` `v2.60` or higher
+-   `shx` `v0.3.4` or higher
 
 ## Building
 
-Simply run `build.js` in the repository root.
+Simply run `./build.js` in the repository root.
+
+## How to setup a feature to build with podjs
+
+To create a polyPod compatible feature, you will need to use this module.
+Follow these instructions to do so:
+
+1.  In order to point to `podjs` from the polypoly-eu repository,
+    add as a `development dependency` on your `package.json` the following:
+
+    ```json
+
+       "devDependencies": {
+          "@polypoly-eu/podjs": "file: <path-that-points-to-platform/podjs>",
+       },
```

Comment:
Two nits about this one:
1. Below, you have specified adding these dependencies with `npm i`. It's probably sensible to do the same here. Alternatively, you'll need to run `npm i` once the package.json file is edited.
2. I'm not totally sure it would be a dev dependency. Since this effectively includes all polyPod functionality, it's very likely that it's a production dependency. Did you check that?

Reply:
for #2: this is how it's right now setup in every feature

Reply:
I'm not sure that's entirely correct, however. But fair enough. I think that it would properly be a *peer* dependency. Let's leave it at that for the time being, then.